app/templates/components/order-card.hbs

```diff
@@ -0,0 +1,38 @@
+<div class="event wide ui grid row">
+  {{#unless device.isMobile}}
+    <div class="ui card three wide computer six wide tablet column">
+      <a class="image" href="#">
+        {{widgets/safe-image src=(if order.event.originalImageUrl order.event.originalImageUrl order.event.originalImageUrl)}}
+      </a>
+    </div>
+  {{/unless}}
+  <div class="ui card thirteen wide computer ten wide tablet sixteen wide mobile column">
+    <a class="main content" href="#">
```

Comment:
The href is `#` right now, since we want to direct the user to the `order.details_after_payment` route which hasn't been implemented yet.